packages/gateway/src/features/registration/schema.graphql

```diff
@@ -527,6 +529,14 @@ type Reinstated {
   registrationStatus: RegStatus
 }
 
+enum Role {
+  ANONYMOUS
+}
+
+directive @auth(requires: Role!) on FIELD_DEFINITION
```

Comment:
There seems to be 2 `auth` directives created? The other one is in common.graphql

Reply:
Good catch!